docs/manual/scala/guide/cluster/Cluster.md

```diff
@@ -56,22 +56,40 @@ Akka Cluster Bootstrap, in Lagom, can be disabled through the property `lagom.cl
 
 #### Akka Discovery
 
-In order to find the peer nodes and form a cluster, Akka Cluster Bootstrap need to be configured to use one of the existing Akka Discovery implementations.
+In order to find the peer nodes and form a cluster, Akka Cluster Bootstrap needs to be configured to use one of the existing Akka Discovery methods:
+
+ 1. Start by choosing one of the methods from [Akka Discovery](https://doc.akka.io/docs/akka/2.6/discovery/) or [Akka Management](https://doc.akka.io/docs/akka-management/1.0/discovery/) as appropriate for your deployment environment. For example, if you are deploying to Kubernetes, the `kubernetes-api` method is recommended. Note that the Akka Discovery method used for Akka Cluster Bootstrap is different than the method used for [[service discovery between services|AkkaDiscoveryIntegration]].
+
+ 2. If you are using one of the Akka Discovery methods provided by Akka Management, you will need to add the library dependency to your project build. Using `kubernetes-api` as an example, in Maven:
+    ```xml
+    <dependency>
+        <groupId>com.lightbend.akka.discovery</groupId>
+        <artifactId>akka-discovery-kubernetes-api_${scala.binary.version}</artifactId>
+        <version>1.0.3</version>
+    </dependency>
+    ```
```

Comment:
It doesn't harm but we don't add Maven-specific instructions in any other `scala/` page of the docs.

Reply:
We do at the top of this one.

Reply:
Should I remove them both?

Reply:
I just did a search and found a few Scala pages that list Maven dependencies:

- https://www.lagomframework.com/documentation/1.5.x/scala/Cluster.html#Dependency
- https://www.lagomframework.com/documentation/1.5.x/scala/PersistentEntityCassandra.html#Project-dependencies
- https://www.lagomframework.com/documentation/1.5.x/scala/PersistentEntityRDBMS.html#Project-dependencies
- https://www.lagomframework.com/documentation/1.5.x/scala/Logging.html

So, I guess this is inconsistent right now, but this pull request is probably not the place to try to resolve the inconsistency. For now, I'll just remove this instance and leave the others for another time.